nlptest/nlptest.py

```diff
@@ -138,6 +142,9 @@ def __init__(
             else:
                 self._config = self.configure(
                     self.DEFAULTS_CONFIG['hubs'][hub])
+        elif task=="translation" :
+            self._config = self.configure(
+                    self.DEFAULTS_CONFIG['task'][task+"-"+hub])
```

Comment:
Aren't we gonna get a `KeyError` for hubs that are different than "huggingface" and "johnsnowlabs"?